config/routes.rb

```diff
@@ -25,8 +25,8 @@
   end
   
   resources :pets, only: [:new, :create, :show, :edit, :update] do
+    resources :weights, only: [:index, :new, :create], path: 'weight'
```

Comment:
```suggestion
    resources :weights, only: [:index, :new, :create]
```

Se tirar o `path` para usar o default do `resouce`, vai quebrar as rotas? Se alterar só a nomenclatura, acho importante trocar nas rotas.